charts/elasticsearch/templates/es-ingress.yaml

```
@@ -1,28 +1,36 @@
 ########################################
 ## Elasticsearch Logging Ingress ##
 ########################################
-{{- if eq .Values.global.plane.mode "data" }}
+{{- if or (eq .Values.global.plane.mode "data") (eq .Values.global.plane.mode "unified") }}
 kind: Ingress
 apiVersion: {{ template "apiVersion.Ingress" . }}
 metadata:
   name: {{ template "elasticsearch.fullname" . }}-ingress
   labels:
-    tier: elasticsearch-networking
-    component: elasticsearch-logging-ingress
+    tier: logging
+    component: {{ template "elasticsearch.name" . }}-logging-ingress
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
     plane: {{ .Values.global.plane.mode }}
   annotations:
+    {{- if .Values.global.authSidecar.enabled  }}
+{{- toYaml .Values.global.extraAnnotations | nindent 4 }}
+    {{- else }}
+    {{- if .Values.global.extraAnnotations }}
+{{- toYaml .Values.global.extraAnnotations | nindent 4 }}
+    {{- end }}
```

Because of `{{- ... nindent` these `toYaml` can be indented however we like.
```suggestion
    {{- if .Values.global.authSidecar.enabled  }}
      {{- toYaml .Values.global.extraAnnotations | nindent 4 }}
    {{- else }}
    {{- if .Values.global.extraAnnotations }}
      {{- toYaml .Values.global.extraAnnotations | nindent 4 }}
    {{- end }}
```